### PR TITLE
Update Hero component layout and font loading

### PR DESCRIPTION
**Pull request by lucaoskaique**
This pull request includes several changes to the Hero component. The layout has been improved and lazy loading has been added for images. Additionally, the font loading in the globals.css file has been updated. The padding and font size in the TailwindCSS config have also been modified. These changes aim to enhance the overall appearance and performance of the Hero component.

### CODE REVIEW

File: src/templates/Home/index.tsx
Comment (VFGarciaDev) [inline src/templates/Home/index.tsx:1]: Ficou faltando adicionar a prop 'donatePage' do componente Hero

File: src/templates/Base/index.tsx [hunk 1]
```diff
@@ -1,21 +1,32 @@
...
+      <Hero
+        bgUrl={Herotemp.bgHero.src}
+        home={{
+          logo: Herotemp.home.logo,
+          title: Herotemp.home.title
+        }}
+      />
 
-    <div className="flex-1">{children}</div>
+      <main className="container mx-auto min-h-screen px-4">{children}</main>
```
Comment (VFGarciaDev) [inline src/templates/Base/index.tsx:24]: Pelo fato de transferir o Hero Componente para componente Base, o style 'min-h-screen' quebra o layout na página de Donate. Testei e achei melhor sem o min-h, oq acha?

File: src/templates/Home/index.tsx
Comment (lucaoskaique) [reply src/templates/Home/index.tsx:1]: Ele nao ta nessa pagina mais

File: src/templates/Base/index.tsx (hunk 1 above)
Comment (lucaoskaique) [reply src/templates/Base/index.tsx:24]: nao quebrou nada aqui, continua com o container fixo no bg e nao só no conteudo

**Pull request merged**